integration-tests/src/tests/nearcore/rpc_nodes.rs

```diff
@@ -443,15 +443,20 @@ fn test_check_unknown_tx_must_return_error() {
 
         let client = new_client(&format!("http://{}", rpc_addrs[0]));
         let tx_hash = transaction.get_hash();
-        let bytes = borsh::to_vec(&transaction).unwrap();
 
         spawn_interruptible(async move {
             loop {
                 let res = view_client.send(GetBlock::latest().with_span_context()).await;
                 if let Ok(Ok(block)) = res {
                     if block.header.height > 10 {
                         let _ = client
-                            .EXPERIMENTAL_tx_status(to_base64(&bytes))
+                            .EXPERIMENTAL_tx_status(RpcTransactionStatusRequest {
+                                transaction_info: TransactionInfo::TransactionId {
+                                    tx_hash,
+                                    sender_account_id: transaction.transaction.signer_id,
+                                },
+                                wait_until: TxExecutionStatus::None,
```

Comment:
With the new changes, you need to say explicitly that you are not waiting for this transaction to appear